packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@okta/courage": "4.4.0",
+    "@okta/courage": "4.6.0-alpha.2029.ga661856",
```

Comment:
Yeah, I was wondering about that myself...

Reply:
In term of functionality shouldn't matter since it is an internal dependency, but we might want to keep the convention since it's public code, like for i18n.

Reply:
shall be beta version and we dont have a good story regarding using release version.

Reply:
Please see the new PR: https://github.com/okta/okta-signin-widget/pull/618